src/modules/transaction/components/Multisignature/Multisignature.js

```diff
@@ -7,20 +7,63 @@ import { txStatusTypes } from '@transaction/configuration/txStatus';
 import { getErrorReportMailto } from 'src/utils/helpers';
 
 import copyToClipboard from 'copy-to-clipboard';
-import { toTransactionJSON, downloadJSON, joinModuleAndCommand } from '@transaction/utils';
+import { downloadJSON, joinModuleAndCommand, toTransactionJSON } from '@transaction/utils';
 import { MODULE_COMMANDS_NAME_MAP } from '@transaction/configuration/moduleCommand';
 import Icon from 'src/theme/Icon';
+import { useAccounts, useCurrentAccount } from '@account/hooks';
+import { calculateRemainingAndSignedMembers, truncateAddress } from '@wallet/utils/account';
+import { ReactComponent as SwtichIcon } from '@setup/react/assets/images/icons/switch-icon.svg';
```

Comment:
Typo in `SwtichIcon`

Reply:
Nice, fixed in two places https://github.com/LiskHQ/lisk-desktop/pull/5366/commits/1c1aad5c9bf10fff1cccc1ebef8b2eeb53c1258a